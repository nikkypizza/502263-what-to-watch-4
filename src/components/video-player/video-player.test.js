import React from 'react';
import renderer from 'react-test-renderer';

import VideoPlayer from './video-player.jsx';
import promoObj from '../../mocks/promo.js';

describe(`VideoPlayer Component`, () => {
  it(`renders`, () => {
    const tree = renderer.create(
        <VideoPlayer
          src={promoObj.trailerUrl}
          poster={promoObj.mainPosterUrl}
          active={true}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
