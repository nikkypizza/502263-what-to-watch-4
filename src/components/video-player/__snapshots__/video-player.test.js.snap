// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer Component renders 1`] = `
<video
  height="175"
  poster="img/the-grand-budapest-hotel-poster.jpg"
  src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
  width="280"
/>
`;
